circuit_knitting/cutting/cutting_evaluation.py

```diff
@@ -246,7 +239,6 @@ def _generate_cutting_experiments(
 ) -> tuple[
     list[QuantumCircuit] | dict[str | int, list[QuantumCircuit]],
     list[tuple[float, WeightType]],
-    list[list[list[QuantumCircuit]]],
```

Comment:
Since the legacy structure isn't needed anymore, the next PR will move this function to the correct module, `cutting_experiments.py`.

We will leave this in place for this PR, as the focus is getting execute-->reconstruct data structures aligned -- not moving large amounts of code.

Does this sound reasonable, @garrison ?

Reply:
Thanks -- and yes, that sounds perfect.